composites/AlgoliaSearch/SearchResults.js

```diff
@@ -0,0 +1,202 @@
+import React from "react";
+import PropTypes from "prop-types";
+import a11ySpeak from "a11y-speak";
+import { injectIntl, intlShape, defineMessages } from "react-intl";
+import styled from "styled-components";
+import { ZebrafiedListTable } from "../basic/Table/ListTable";
+import { Row } from "../basic/Table/Row";
+import colors from "../../style-guide/colors.json";
+
+const messages = defineMessages( {
+	noResultsText: {
+		id: "searchresults.noresultstext",
+		defaultMessage: "No results found.",
+	},
+	foundResultsText: {
+		id: "searchresults.foundresultstext",
+		defaultMessage: "Found {resultsCount} { resultsCount, plural, one {result} other {results} }",
+	},
+} );
+
+/**
+ * The title of the search result item.
+ */
+const SearchResultTitle = styled.p`
+	padding-left: 10px;
+	margin: 0;
+	font-size: 1em;
+	font-weight: normal;
+`;
+
+/**
+ * The link to the search result.
+ */
+const SearchResultLink = styled.a`
+	color: ${ colors.$color_black };
+	padding: 10px;
+	
+	&:hover, &:focus {
+		color: ${ colors.$color_pink_dark };
```

Comment:
for a11y, a focus indication with color alone is not sufficient. For now, this keeps the native outline styles so it's OK. But, if the outline is going to be reset in the implementation, then we'd need to add something else for focus.